pillar.example

```diff
@@ -78,13 +78,16 @@ salt:
   # salt cloud config
   cloud:
     master: salt
-    folders:
-      - cloud.providers.d/key
-      - cloud.profiles.d
-      - cloud.maps.d
-    providers:
-      - ec2
-      - gce
+
+    # You can take profile and map templates from an alternate location
+    # if you want to write your own.
+    template_sources:
+      providers: salt://templates/cloud.providers.d
+      profiles: salt://templates/cloud.profiles.d
+      maps: salt://templates/cloud.maps.d
```

Comment:
pillar.example generally follows the default. You should prefix these or something (salt://salt/files/cloud.maps.d would match what minion.sls and master.sls already do).